lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
Otoh, if all required (or optional -- don't think tagging both is needed) parameters were tagged in the Plugin class, validators could use that to switch logic accordingly...